app/src/main/java/com/randomappsinc/studentpicker/grouping/GroupMakingSettingsViewHolder.java

```diff
@@ -70,4 +75,35 @@ private void applyNumberOfGroups() {
         }
         numGroups.clearFocus();
     }
+
+    @Override
+    public void beforeTextChanged(CharSequence s, int start, int count, int after) {}
+
+    @Override
+    public void onTextChanged(CharSequence s, int start, int before, int count) {}
+
+    @Override
+    public void afterTextChanged(Editable editable) {
+        if (isEditableAndFocused(editable, namesPerGroup)) {
+            numGroups.setText(getOffset(namesPerGroup));
+        } else if (isEditableAndFocused(editable, numGroups)) {
+            namesPerGroup.setText(getOffset(numGroups));
+        }
+    }
+
+    private String getOffset(EditText editText) {
```

Comment:
It is weird that we do things on empty input. When the input is empty or 0, I think we should just do nothing. We should only act if the enter in a number that's larger than 1.